bool PHOTO=true
bool VIDEO=false

string img_name
bool imagen_tipe
string address

---------------
string dado
